CODE_OF_CONDUCT.md

```diff
@@ -48,17 +48,17 @@ threatening, offensive, or harmful.
 This Code of Conduct applies both within project spaces and in public spaces
 when an individual is representing the project or its community. Examples of
 representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
+address, posting via an official social media account or acting as an appointed
```

Comment:
We use the serial comma (aka the Oxford or Harvard comma) so I think this comma should be put back in.

Reply:
Added back